modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/core/schema/DataLayout.es.js

```diff
@@ -56,6 +56,10 @@ export class DataLayoutSchema extends Schema {
 		return this[SYMBOL_RAW].rules;
 	}
 
+	get dataLayoutFields() {
+		return this[SYMBOL_RAW].dataLayout.dataLayoutFields;
+	}
```

Comment:
This is not a complete implementation of this, we need to merge the fields within the `dataLayoutFields` with the fields that are within the `pages` structure. This is because the `dataLayoutFields` contains fields that are not within the `pages`, they are treated as fields at the object level but the fields within the `pages` at the form view level.